docs/en/design/visualization.md

```diff
@@ -1,3 +1,76 @@
 # Visualization
 
-Coming soon. Please refer to [chinese documentation](https://mmengine.readthedocs.io/zh_CN/latest/design/visualization.html).
+## 1 Overall Design
+
+Visualization provides an intuitive explanation of the training and testing process of the deep learning model. In OpenMMLab, we expect the visualization function to meet the following requirements:
+
+- Provides rich out-of-the-box features that can meet most computer vision visualization tasks.
+- High scalability through easy extension and customization, as visualization functions are usually highly diversified.
+- Able to visualize at any point in the training and testing process.
+- A group of unified APIs to all OpenMMLab libraries, which is convenient for users to understand and maintain.
+
+Based on the above requirements, we introduced the `Visualizer` and various `VisBackend` such as `LocalVisBackend`, `WandbVisBackend`, and `TensorboardVisBackend` in OpenMMLab 2.0. The visualization functions work not only on the image data but also things like configurations, scalars, and model diagrams.
+
+- For convenience, the APIs provided by the `Visualizer` implement the drawing and storage functions. `VisBackend`, as an internal property of `Visualizer`, will be called by `Visualizer` when needed to store data to different backends.
+- Considering that you may want to store multiple backends after drawing, `Visualizer` can be configured with various VisBackends. When the user calls the storage API of the `Visualizer`, it will traverse and call all the specified APIs of `VisBackend` internally.
```

Comment:
```suggestion
- Considering that you may want to write data to multiple backends after drawing, `Visualizer` can be configured with multiple backends. When the user calls the storage API of the `Visualizer`, it will traverse and call all the specified APIs of `VisBackend` internally.
```